torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -613,6 +615,10 @@ void Reducer::set_logger(std::weak_ptr<c10d::Logger> logger) {
 // This function is only to be called from the autograd thread.
 void Reducer::autograd_hook(size_t index) {
   std::lock_guard<std::mutex> lock(this->mutex_);
+  if (!first_autograd_hook_called_) {
+    first_autograd_hook_called_ = true;
+    num_bwd_calls_++;
```

Comment:
Curious: In PyTorch, do people care about `++num_bwd_calls_;` instead of `num_bwd_calls_++`?

Reply:
To me here it doesn't make a difference because increment pre/postfix doesn't matter here, right?

Reply:
I guess I mean for the minor difference of the temporary created for `num_bwd_calls_++`. I am not sure if our default compiler settings optimize that.